docker/e2e/Makefile

```diff
@@ -81,5 +81,10 @@ start-docker-compose: start-nodes
 start-nodes-background: genfiles/build-abci genfiles/build-ledger genfiles/generate-tendermint-e2e-config genfiles/docker-compose.json
 	docker-compose -f genfiles/docker-compose.json -p e2e up --detach
 
+start-node-background: genfiles/build-abci genfiles/build-ledger genfiles/generate-tendermint-e2e-config genfiles/docker-compose.json
```

Comment:
To avoid confusion, can we use `start-single-node-background` instead? The name threw me off.